pyo3-derive-backend/src/pymethod.rs

```diff
@@ -62,6 +63,23 @@ fn check_generic(sig: &syn::Signature) -> syn::Result<()> {
     Ok(())
 }
 
+pub fn gen_py_const(
+    cls: &syn::Type,
+    name: &syn::Ident,
+    attrs: &mut Vec<syn::Attribute>,
+) -> syn::Result<Option<TokenStream>> {
+    let spec = ConstSpec::parse(name, attrs)?;
+    if spec.is_class_attr {
+        let wrapper = quote! {
+            fn __wrap(py: pyo3::Python<'_>) -> pyo3::PyObject {
+                pyo3::IntoPy::into_py(#cls::#name, py)
```

Comment:
I think this will only work for `Copy` types, but I guess that's pretty much everything that could be generated by `const fn` anyway. Might want to document this restriction though?

Reply:
Associated `const`, like any other `const` variables, are basically syntactically replaced by their compile-time evaluated expression everywhere they appear.

As a result, you can do:
```rust
const FOO: String = String::new();

fn foo(x: String) {}

fn main() {
    foo(FOO);
    foo(FOO);
}
```
although `String` is not copy.

Reply:
Oh yes, of course, my bad! 😄